views/blocks/payment_other_with_logo.tpl

```diff
@@ -0,0 +1,55 @@
+[{*
+Uses the content of original OXID payment_other.tpl
+Rendering logo was needed but could not be achieved by extending the block
+*}]
+<dl>
+    <dt>
+        [{assign var="oShopConf" value=$oViewConf->getConfig()}]
+        <input id="payment_[{$sPaymentID}]" type="radio" name="paymentid" value="[{$sPaymentID}]" [{if $oView->getCheckedPaymentId() == $paymentmethod->oxpayments__oxid->value}]checked[{/if}]>
+        [{* Logo added *}]
+        <img src="[{$oShopConf->getShopUrl()}]modules/wirecard/paymentgateway/out/img/[{ $paymentmethod->oxpayments__wdoxidee_logo->value }]">
```

Comment:
I'd suggest adding a method to the Payment model for getting the logo URL (e.g. `getLogoUrl()`). That way we don't have any hardcoded paths in the view.

Reply:
Logo code removed from this PR